src/components/molecules/AdminFooter/AdminFooter.tsx

```diff
@@ -0,0 +1,32 @@
+const navigation = [
+  { name: "About", href: "#about" },
+  { name: "Blog", href: "#blog" },
+  { name: "Jobs", href: "#jobs" },
+  { name: "Press", href: "#press" },
+  { name: "Accessibility", href: "#accessibility" },
+  { name: "Partners", href: "#partners" },
```

Comment:
Are these used to navigate inside the page? 
Which are the components that have anchors and/or ids? 

Seems like there either should be no `href` or these should be shared constants (maybe even created by `generateId()`) in some `./src/settings/` file

Reply:
I guess they're going to be external URLs. Added consts to `urlSettings`: https://github.com/sparkletown/sparkle/pull/2812/commits/0f78efe3a381196c9e5b7cacf68aa5c14daa88bc